test/size_test.go

```diff
@@ -0,0 +1,195 @@
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/fsouza/go-dockerclient"
+	"github.com/kr/pretty"
+	"github.com/stretchr/testify/assert"
+)
+
+type rockerBuildFn func(string, io.Writer, ...string) error
+
+func runImageSizeTestWithDockerVersion(t *testing.T, dockerVersion string) {
+
+	cleanup, rocker, err := setupDockerVersionedEnv(t, dockerVersion)
+	assert.Nil(t, err, "setup env")
+
+	defer cleanup()
+
+	time.Sleep(2 * 1e9)
+
+	fmt.Fprintf(os.Stdout, "docker env ready, building test image\n")
```

Comment:
things like these should be wrapped in some logging helper, respecting verbosity level, ofc